easybuild/main.py

```diff
@@ -684,7 +684,7 @@ def main(args=None, logfile=None, do_build=None, testing=False, modtool=None, pr
         options.list_prs,
         options.merge_pr,
         options.review_pr,
-        options.terse,
+        options.terse and not options.missing_modules,
```

Comment:
Why is early stop disabled for `--terse --missing-modules`?

Shouldn't `silent` be controlled by `options.terse` rather than being set to `True` in `clean_exit` below, as is done at the end of `main`?

Reply:
Oh, I see, it's because `--missing-modules` is only acted upon in `process_eb_args`, which is called after this.

OK, fine, but then we should add a comment here to explain why this is done like this, because it looks out of place:

```suggestion
        # --missing-modules is processed by process_eb_args,
        # so we can't exit just yet here if it's used in combination with --terse
        options.terse and not options.missing_modules,